package.json

```diff
@@ -81,7 +82,8 @@
     "coverageDirectory": "coverage",
     "coverageReporters": [
       "text",
-      "html"
+      "html",
+      "lcov"
```

Comment:
Wouldn't we only need lcov in this list?

Reply:
We  could, but I personally use the HTML reporter locally. LCOV is not human friendly.